feat(examples/starter): add example starter setup (#200)
commercetools/merchant-center-application-kit: examples/starter/README.md

<p align="center">
  <a href="https://commercetools.com/">
    <img alt="commercetools logo" src="http://cdn.rawgit.com/commercetools/press-kit/master/PNG/72DPI/CT%20logo%20chrom%20black%20horizontal%20RGB%2072dpi.png">
  </a>
  <b>Merchant Center Examples: Starter.</b>
</p>

> This example contains the bare minimum setup to get you started with the development of the Merchant Center Application.

Reviewer: ```suggestion
This example application gives you the minimum setup to develop and launch a working Merchant Center application.
```

## Installation

Checkout the github repository and copy-paste the `examples/starter` folder into your new project.

> 

Reviewer: ```suggestion

To do so, run the following commands:

```

```bash
$ git checkout git@github.com:commercetools/merchant-center-application-kit.git

$ mkdir my-new-custom-application-project
$ cp -R merchant-center-application-kit/examples/starter/* my-new-custom-application-project

> 

Reviewer: ```suggestion

Next, run the following command to initialize the yarn workspace:

```

$ cd my-new-custom-application-project
$ yarn
```

## Start the development server

> 

Reviewer: ```suggestion

Run the following command to start the development server and launch the application:

```
